src/test/kotlin/apply/application/EvaluationServiceTest.kt

```diff
@@ -0,0 +1,108 @@
+package apply.application
+
+import apply.domain.evaluation.Evaluation
+import apply.domain.evaluation.EvaluationRepository
+import apply.domain.recruitment.Recruitment
+import apply.domain.recruitment.RecruitmentRepository
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+import org.mockito.ArgumentMatchers.anyLong
+import org.mockito.BDDMockito.given
+import org.mockito.BDDMockito.willDoNothing
+import org.mockito.Mockito.mock
+import org.springframework.beans.factory.annotation.Autowired
+import support.createLocalDateTime
+import java.util.Optional
+
+internal class EvaluationServiceTest {
+    @Autowired
+    private lateinit var evaluationService: EvaluationService
+    private val recruitmentRepository: RecruitmentRepository = mock(RecruitmentRepository::class.java)
+    private val evaluationRepository: EvaluationRepository = mock(EvaluationRepository::class.java)
+    private lateinit var recruitments: List<Recruitment>
+    private lateinit var preCourseEvaluation: Evaluation
+    private lateinit var firstEvaluation: Evaluation
+    private lateinit var secondEvaluation: Evaluation
+    private lateinit var evaluations: List<Evaluation>
+
+    @BeforeEach
+    internal fun setUp() {
+        evaluationService = EvaluationService(evaluationRepository, RecruitmentService(recruitmentRepository))
+
+        recruitments = listOf(
+            Recruitment(
+                "웹 백엔드 2기",
+                false,
+                createLocalDateTime(2019, 10, 25, 10),
+                createLocalDateTime(2019, 11, 5, 10)
+            ),
+            Recruitment(
+                "웹 백엔드 3기",
+                true,
+                createLocalDateTime(2020, 10, 25, 15),
+                createLocalDateTime(2020, 11, 5, 10)
+            )
+        )
+
+        preCourseEvaluation = Evaluation(
+            "프리코스 대상자 선발",
+            "[리뷰 절차]\n" +
+                "https://github.com/woowacourse/woowacourse-docs/tree/master/precourse",
+            1L,
+            0L,
+            1L
```

Comment:
동일한 자료형이 두 번 이상 반복되는 경우 인자의 이름을 지정하면 어떨까요?

```suggestion
            recruitment = 1L,
            beforeEvaluation = 0L,
            id = 1L
```

Reply:
확실히 명시적인게 좋네요.

저도 제이슨처럼 이런 상황에서는 명시해주는 습관을 길러보겠습니다.

좋은 기능은 활용해야죠!